java-tools/OsmAndServer/src/main/java/net/osmand/server/controllers/pub/SearchController.java

```diff
@@ -160,4 +157,20 @@ public ResponseEntity<String> getWikiImages(@RequestParam String northWest, @Req
         FeatureCollection collection = wikiService.getImages(northWest, southEast);
         return ResponseEntity.ok(gson.toJson(collection));
     }
+    
+    @GetMapping(path = {"/get-poi-by-coords"}, produces = "application/json")
+    @ResponseBody
+    public ResponseEntity<String> getPoiByCoords(@RequestParam double lat,
```

Comment:
No osm type specified, it's likely possible bug cause osmid could be way/node/relation